client/components/videos-ui/style.scss

```diff
@@ -63,7 +63,7 @@
 	.videos-ui__body {
 		max-width: 1280px;
 		margin: 0 auto;
-		padding: 64px 20px 41px;
+		padding: 64px 20px 97px;
```

Comment:
This is so that when scrolling to the bottom, the last video section isn't partially hidden behind the new fixed footer.